packages/js/src/notifications/notifications.ts

```diff
@@ -242,4 +242,12 @@ export class Notifications extends BaseModule {
       })
     );
   }
+
+  clearCache({ filter }: { filter?: NotificationFilter } = {}): void {
+    if (filter) {
+      return this.#notificationsCache.clear(filter ?? {});
```

Comment:
Should this mean `{}` or `{archived: false}`? I'm guessing the first but just checking.

Reply:
the first ;)